### PR TITLE
Remove existing coverage goals

### PR DESCRIPTION
**Pull request by peterschrammel**
This is obsolete in cover.cpp as the --existing-coverage
option will be implemented differently.

Companion: https://github.com/diffblue/test-gen/pull/1325

### CODE REVIEW

**Review by chrisr-diffblue — APPROVED** [review]
Seems reasonable to me.

File: src/goto-instrument/cover.cpp [hunk 1]
```diff
@@ -1325,10 +1188,9 @@ void instrument_cover_goals(
             basic_blocks.source_location_of(block_nr);
 
           // check whether the current goal already exists
```
Comment (mgudemann) [inline src/goto-instrument/cover.cpp]: the comment doesn't apply any more

**Review by mgudemann — APPROVED** [review]
LGTM but the comment should be adapted

**Comment by chrisr-diffblue** [comment]
@mgudemann - Thanks for spotting the outdated comment. As @peterschrammel is on holiday at the moment I've taken the liberty of adjusting the comment. Could you take another look and see if you are happy with my change to the comment? If you are could you either merge or let me know and I'll merge.

**Review by mgudemann — APPROVED** [review]

**Pull request merged**